sqlitecluster/SQLiteNode.h

```diff
@@ -278,6 +278,8 @@ class SQLiteNode : public STCPManager {
 
     void _dieIfForkedFromCluster();
 
+    void _processPeerMessages(uint64_t& nextActivity, SQLitePeer* peer, bool unlimited = false);
```

Comment:
NAB but why not make unlimited mandatory?

Reply:
No particular reason I guess, I feel like we often supply defaults for things like this but I don't see any particular reason to do or not to do it here.